config/webpack.dev.js

```diff
@@ -52,7 +52,18 @@ module.exports = function (options) {
     plugins: [
       new DashboardPlugin(),
       extractCSS,
```

Comment:
The declaration for this seems to have been commented. Is this correct?

Reply:
fixed - thanks for the catch!